docs/knn/index.md

```diff
@@ -125,6 +125,34 @@ GET my-knn-index-1/_search
 }
 ```
 
+The formulas we use to perform calculations are as follows:
```

Comment:
So I don't love the double : pattern here. How do you feel about removing this sentence and just doing the following?

Euclidian distance formula:

Cosine similarity formula:

Reply:
That does look cleaner, so I'll go with your suggestion.